src/vocgui/serializers.py

```diff
@@ -174,10 +173,6 @@ def get_total_discipline_children(self, obj):
         :return: sum of children
         :rtype: int
         """
-        queryset = Discipline.objects.filter(
-            Q(released=True)
-            & Q(created_by=obj.id)
-            & Q(id__in=get_valid_discipline_ids())
-        )
-        queryset = [obj for obj in queryset if obj.is_root_node()]
+        queryset = Discipline.objects.filter(Q(released=True) & Q(created_by=obj.id))
```

Comment:
```suggestion
        queryset = Discipline.objects.filter(released=True, created_by=obj.id)
```
I'm not 100% sure, but is `&` just a regular `AND`-operator? If so, I think you don't need `Q` objects for this...